pyccel/ast/type_annotations.py

```diff
@@ -43,15 +43,11 @@ class VariableTypeAnnotation(PyccelAstNode):
     ----------
     class_type : PyccelType
         The requested Python type of the variable.
-
-    is_const : bool, default=False
-        True if the variable cannot be modified, false otherwise.
     """
-    __slots__ = ('_class_type', '_is_const')
+    __slots__ = ('_class_type',)
     _attribute_nodes = ()
-    def __init__(self, class_type : PyccelType, is_const : bool = False):
+    def __init__(self, class_type : PyccelType):
```

Comment:
Could you please add a blank line before the `__init__`?
```suggestion
    __slots__ = ('_class_type',)
    _attribute_nodes = ()

    def __init__(self, class_type : PyccelType):
```